### ISSUE TITLE
Ghosting files

### ISSUE THREAD

**Issue by holantomas** [haiwen/seadrive-gui#38]
Hi, by alot of reports on forum a created this issue. This issue has two variant which looks same but way are diffrent. So there's a problem with Folders(Never saw file but user's on forum report them) that are appear in Seafile with no change date and no files. Sometimes there's only empty subfolders.

**Variant # 1**
Folders appears without user interaction(maybe modified files inside). If we delete this empty folder, it appers again after some hours.
```
 - / #root of library
   - some_dir/
     - 20171101_example_dir/ # this dir is normally used and contains files
     - ... #some other folders and files
   - 20171101_example_dir/ # this dir is created automaticly with no files and date change 1970-01-01
   - ... #some other folders and files
```

**Variant # 2**
Same as before, empty folder with empty subfolders. This happend after folder rename. So if we rename folder `Test` to `Test - test` then is folder `Test` automatically re-created empty with only empty subfolders. But this happend on same folder level no only in root

**Comment by killing** [comment]
I think both variants are actually the same problem: the folder was renamed or moved. We've reproduced this issue and build a fix. Will make it available for test next Monday.

**Comment by freeplant** [comment]
This problem is fixed in version 0.8.3.
